modules/ducktests/tests/ignitetest/services/ignite_app.py

```diff
@@ -46,14 +46,17 @@ def __init__(self, context, config, java_class_name, num_nodes=1, params="", sta
         self.servicejava_class_name = servicejava_class_name
         self.java_class_name = java_class_name
         self.params = params
+        self.start_ignite = start_ignite
```

Comment:
An incorrect place to store this flag.
The flag is never used inside the class.
Should be stored at IgniteAwareService.

Reply:
it is used in await_started, string 52

Reply:
As I said, de-facto, this usage can and *should* be encapsulated at `IgniteAwareService`.`await_started()`